### ISSUE TITLE
error in nanopub typing for reference publication

### ISSUE THREAD

**Issue by micheldumontier** [MaastrichtU-IDS/knowledge-collaboratory#13]
Looks like there is a problem in the nanopub in reference to the publication, namely the use of the predicate dct:type has a literal value of "dctypes"

```biolink:id "collaboratory:PUBCHEM.COMPOUND:5073-biolink:treats-MONDO:0005090" ;
    biolink:publications <https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=c0c3eeb6-8a75-0b20-2008-396e63cddcdb> .
  <https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=c0c3eeb6-8a75-0b20-2008-396e63cddcdb> dct:type "dctypes" ;
    a biolink:Publication ;
```

**Comment by vemonet** [comment]
It is not a bug, it is a feature!

The BioLink model requires to define a type for publication, and is using `dct:type`, pointing to a restricted string (which should be one of:

```yaml
 values_from: 
    - dctypes
    - fabio
    - MESH_PUB
    - COAR_RESOURCE
    - WIKIDATA 
```

More about it in the Publication doc page: https://biolink.github.io/biolink-model/docs/Publication.html

> Description: Ontology term for publication type may be drawn from Dublin Core types (https://www.dublincore.org/specifications/dublin-core/dcmi-type-vocabulary/), FRBR-aligned Bibliographic Ontology (https://sparontologies.github.io/fabio/current/fabio.html), the MESH publication types (https://www.nlm.nih.gov/mesh/pubtypes.html), the Confederation of Open Access Repositories (COAR) Controlled Vocabulary for Resource Type Genres (http://vocabularies.coar-repositories.org/documentation/resource_types/), Wikidata (https://www.wikidata.org/wiki/Wikidata:Publication_types), or equivalent publication type ontology. When a given publication type ontology term is used within a given knowledge graph, then the CURIE identified term must be documented in the graph as a concept node of biolink:category biolink:OntologyClass.

It was actually creating issues with the SHACL validation, but the pull request to fix it has been merged recently, so that should not be a problem once I update the Collaboratory to the latest version of the BioLink SHACL shape: https://github.com/biolink/biolink-model/pull/1218

**Comment by micheldumontier** [comment]
Right, the text is very clear that it should be a CURIE for a *type* that is defined in one of those terminologies, not that it should be one of those terminologies. It's good that you caught the problem and that this will be fixed for the SHACL validation.
